scene/animation/tween.cpp

```diff
@@ -154,6 +154,21 @@ Ref<MethodTweener> Tween::tween_method(const Callable &p_callback, const Variant
 	return tweener;
 }
 
+Ref<SubtweenTweener> Tween::tween_subtween(const Ref<Tween> &p_subtween) {
+	CHECK_VALID();
+
+	Ref<SubtweenTweener> tweener = memnew(SubtweenTweener(p_subtween));
+
+	// Remove the tween from its parent tree, if it has one.
+	// If the user created this tween without a parent tree attached,
+	// then this step isn't necessary.
+	if (tweener->subtween->parent_tree != nullptr) {
```

Comment:
Won't this crash if I call `tween.tween_subtween(null)`?

Reply:
That's a good question - I never thought to try `tween.tween_subtween(null)`, admittedly, but I could see that situation coming up in some use cases where the user isn't sure if some other method is going to return a Tween or not.

For this specific line, I think it'd be sufficient just to check if `tweener->subtween` is `null` or not before the `parent_tree` check, but there may also be other parts of the code where a null subtween needs to be handled, perhaps by treating it like a no-op step in the tween sequence.

Reply:
Yep, it crashes. You can just do `ERR_FAIL_COND_V(p_subtween.is_null(), nullptr);`

Reply:
Thanks! I'll do that for now, since overall people probably shouldn't be passing `null` as a subtween, and then later if it appears to be a situation people are running into for some reason, we can revisit it then :)